app/experimenter/nimbus-ui/src/components/Summary/TableAudience/index.test.tsx

```diff
@@ -4,10 +4,10 @@
 
 import React from "react";
 import { render, screen } from "@testing-library/react";
-import { MockedCache, mockExperimentQuery } from "../../lib/mocks";
-import TableSummary from ".";
-import { getExperiment_experimentBySlug } from "../../types/getExperiment";
-import { NimbusExperimentChannel } from "../../types/globalTypes";
+import { MockedCache, mockExperimentQuery } from "../../../lib/mocks";
+import TableAudience from ".";
```

Comment:
Oh yeah, and while nearly all the changes here were made automatically by VSCode, this is one by-hand change I made: Realized the test passes because it's internally consistent, but `TableSummary` should technically be `TableAudience` here